packages/javascript/package.json

```diff
@@ -2,25 +2,32 @@
   "name": "@livekit/protocol",
   "version": "1.27.1",
   "description": "",
-  "type": "module",
```

Comment:
why not leave that in and make the ending for cjs .cjs like in the node repo?

Reply:
we could do that, yeah. i was just keeping in line with components-js which doesnt use type:module either. your call

Reply:
My vote goes to consistency, we can set type module here and in node sdks and adapt it in components or use the existing components pattern everywhere. No strong opinion either way, just that we use the same pattern would be nice